templates/vsftpd.conf.orig

```diff
@@ -1,118 +0,0 @@
-# Example config file /etc/vsftpd/vsftpd.conf
-#
```

Comment:
Our current template is according to our current production ftp/sftp config setting